src/core/src/op/fake_convert.cpp

```diff
@@ -61,15 +91,40 @@ bool FakeConvert::visit_attributes(ov::AttributeVisitor& visitor) {
 }
 
 void FakeConvert::validate_type() const {
-    const auto& valid_types = fake_convert::get_valid_types();
-    OPENVINO_ASSERT(std::find(valid_types.begin(), valid_types.end(), m_destination_type) != valid_types.end(),
-                    "Bad format for f8 conversion type: " + m_destination_type);
+    const auto& valid_types = fake_convert_details::get_valid_types();
```

Comment:
Use reference implementation to get type?

Reply:
This is a check of the `destination_type` attribute, called to validate it during validate_and_infer_types. 
I'm not convinced to the need of involving reference implementation for that, please share more details of this suggestion or we can discuss it offline.